pkg/apis/minio.min.io/v2/helper.go

```diff
@@ -819,12 +819,10 @@ func (z *Pool) Validate(zi int) error {
 	if z.Servers*z.VolumesPerServer < 4 {
 		// Erasure coding has few requirements.
 		switch z.Servers {
-		case 1:
```

Comment:
How easy to check for minimal MinIO version before ignoring this ?

Reply:
it's relatively easy @vadmeste as long as they have labels in the format we release, we could check if the image for the tenant is populated and compare

Reply:
should we do it? @dvaldivia